File: Java/Area_Triangle.java
package com.lev;

import java.util.Scanner;

public class Area_Triangle {
    public static void main(String[] args) {
        System.out.print("Enter 3 sides : ");
        Scanner in = new Scanner(System.in);
        int a = in.nextInt();
        int b = in.nextInt();
        int c = in.nextInt();
        float s = (a + b + c)/2;
        float area =(float) Math.sqrt(s*(s-a)*(s-b)*(s-c));
        System.out.print("Area : "+area);
    }
}

File: Python/Area_Triangle.py
import math
print("Enter 3 sides :",end='')
a=int(input())
b=int(input())
c=int(input())
s=(a+b+c)/2.0;
area=math.sqrt(s*(s-a)*(s-b)*(s-c))
print("Area :",float(area))
